blocks/bricks/attention.py

```diff
@@ -290,8 +290,8 @@ class SequenceContentAttention(GenericSequenceAttention, Initializable):
     match_dim : int
         The dimension of the match vector.
     state_transformer : :class:`.Brick`
-        A prototype for state transformations. If ``None``, the default
-        transformation from :class:`.Parallel` is used.
+        A prototype for state transformations. If ``None``,
+        a linear transformation is used.
```

Comment:
The issue of backward compatibility is a big issue with this PR and this is just one example.
What will happen to code outside this repository? What about blocks-extra and examples?